static/app/utils/replays/icons.tsx

```diff
@@ -0,0 +1,37 @@
+import {generateIconName} from 'sentry/components/events/contexts/utils';
+
+const PLATFORM_ALIASES = {
+  ios: 'apple',
+  mac: 'apple',
+  macos: 'apple',
+  'mac-os-x': 'apple',
+  darwin: 'apple',
+  tvos: 'apple-tv',
+  watchos: 'apple-watch',
+  iphone: 'apple-iphone',
+  ipad: 'apple-ipad',
+  'legacy-edge': 'edge-legacy',
+  'mobile-safari': 'safari',
+  'chrome-mobile-ios': 'chrome',
+  'google-chrome': 'chrome',
+  'chrome-os': 'chrome',
+  net: 'dotnet',
+  'net-core': 'dotnetcore',
+  'net-framework': 'dotnetframework',
```

Comment:
how did you test which platforms needed aliases??

Reply:
It was manual - I used ctrl-F to check the contextIcon stories vs the svg/ folder in the platformicons repo

Reply:
not the manual labor✊

Reply:
not the not the